src/vivarium/framework/state_machine.py

```diff
@@ -354,10 +356,10 @@ def __len__(self):
         return len(self.transitions)
 
     def __str__(self):
-        return str([str(x) for x in self.transitions])
+        return "TransitionSet()"
```

Comment:
do we at least want to include the name in str representations where we may have multiple of the objects?

Reply:
I think we are leaning to little info. The name should be unique and informational. In Jame's words if you are looking at reprs you're holding the object so you can inspect the transitions

I mostly just want to be able to print and not get garbage

Reply:
So the way I think about this is that if you have a typical object that has some methods and attributes that themselves may be complex objects, your str/repr should be informative and succinct (and in the case of the repr help you uniquely identify the object).

If you're hanging on to an object that is some kind of data structure or container (as the transition set is) you want to detail the container type as succinctly as possible but also give an indication of what it contains.

Something like f"TransitionSet(self._state_name, [t.output_state.name for t in self.transitions])"
or similar seems like a good str.